<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="10" total="11" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="78524" started-at="2018-06-28T16:57:41Z" finished-at="2018-06-28T16:58:59Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="78524" started-at="2018-06-28T16:57:41Z" finished-at="2018-06-28T16:58:59Z">
      <class name="com_Milan_Test.CycleListPageTest">
        <test-method status="PASS" signature="Setup()[pri:0, instance:com_Milan_Test.CycleListPageTest@2b4bac49]" name="Setup" is-config="true" duration-ms="77130" started-at="2018-06-28T22:27:41Z" finished-at="2018-06-28T22:28:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="EnabledconditionARTTypeTest()[pri:4, instance:com_Milan_Test.CycleListPageTest@2b4bac49]" name="EnabledconditionARTTypeTest" duration-ms="215" started-at="2018-06-28T22:28:58Z" finished-at="2018-06-28T22:28:58Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at com_Milan_Test.CycleListPageTest.EnabledconditionARTTypeTest(CycleListPageTest.java:154)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EnabledconditionARTTypeTest -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:com_Milan_Test.CycleListPageTest@2b4bac49]" name="Teardown" is-config="true" duration-ms="749" started-at="2018-06-28T22:28:59Z" finished-at="2018-06-28T22:28:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- com_Milan_Test.CycleListPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
